RELEASE.md

```diff
@@ -1,5 +1,20 @@
 # What-If Tool releases
 
+## Release 1.8.0
+
+This version greatly increases the number of datapoints that can be loaded into
+the tool, in both TensorBoard and notebook mode. WIT can now handle on the order
```

Comment:
nit: all notebook modes (or explicitly list jupyterlab jupyter and colab)?

Reply:
changed to "modes"